tutorials/3d/csg_tools.rst

```diff
@@ -8,7 +8,7 @@ shapes or custom meshes to create more complex shapes. In 3D modelling software,
 CSG is mostly known as "Boolean Operators".
 
 Level prototyping is one of the main uses of CSG in Godot. This technique allows
-users to create simple versions of most common shapes by combining primitives.
+users to create the most common shapes by combining primitives.
```

Comment:
This use of simple was actually okay and makes more sense than the edited version. 

For clarity, it is okay to call a triangle a "simple" shape vs. an n-sided polygon, but it is not okay to call something "simple to understand" like "this script is simple"

Reply:
I agree that the original was okay, but IMO it's still clear this way.